main.cpp

```diff
@@ -113,20 +113,20 @@ struct Knife
     void unscrew();
 };
 
-void Knife::Incision::hurt(float pain)
+float Knife::Incision::getHurtLevel()
 {
-    pain = incisionDepth + incisionLength;
+    return depth + length;
 }
 
 void Knife::cut(Incision prettyNastyCut)
 {
-    prettyNastyCut.hurt(1);
+    prettyNastyCut.getHurtLevel();
```

Comment:
If you're going to call this function, since it returns a value, why not print out that value via std::cout?  
```
std::cout << "cut pain level: " << prettyNastyCut.getHurtLevel() << "\n";
```
that'll give you some kind of output when you call `knife.cut(arg)` at some point.
- [ ] revise